xml/geo_manage_hawk2_i.xml

```diff
@@ -0,0 +1,316 @@
+     <step>
+      <para> Enter a custom <guimenu>Cluster name</guimenu> with which to identify the cluster in
+       the <guimenu>Dashboard</guimenu>. </para>
+     </step>
+     <step>
+      <para>Enter the <guimenu>Hostname of a node in the cluster</guimenu>.</para>
+     </step>
+     <step>
+      <para>To specify further options like the update interval for &hawk;2, the port to connect
+       to or if to use HTTPS (default), click <guimenu>Advanced</guimenu> and enter the details.
+      </para>
+     </step>
+     <step>
+      <para>Confirm your changes with <guimenu>Add</guimenu>.</para>
+      
+     </step>
+     <step>
+      <para> To add more clusters to the dashboard, click <guimenu>Add Cluster</guimenu> again and
+       enter the details for the next cluster. </para>
+      <figure>
+       <title>&hawk;2&mdash;Dashboard</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="hawk2-dashboard-multiple-clusters.png" width="100%" format="PNG"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="hawk2-dashboard-multiple-clusters.png" width="80%" format="PNG"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+     </step>
+    </substeps>
+   </step>
+   <step>
+    <para> To remove a cluster from the dashboard, click the <literal>x</literal> icon next to the
+     cluster's name. </para>
+   </step>
+   <step>
+    <para> To view details about a cluster, enter a password and click <guimenu>Connect</guimenu>. </para>
+    <para> &hawk;2 will show you a summary with the status of the respective nodes, resources, and tickets. 
+     <!--
+      (05:20:55 PM) krig: each column is a node except the last one which is for stopped resources, 
+                          and each row is a resource 
+      (05:21:24 PM) krig: the frame of each box is the node status: green is good, red is bad, gray is offline
+      (05:21:54 PM) krig: the fill color of each box is the status of the resource on that node: 
+                          here green means running, blue means master and orange means slave
+      (05:09:42 PM) krig: in the dashboard, pointing at each box gives more information (details)
+      -->
+     <remark>taroth 2015-10-29: krig: is the following still true?</remark> From here, you can
+     administer the cluster with &hawk;2 as usual.</para>
+   </step>
+  </procedure>
+   <para>
+    Any status changes for nodes or resources are reflected almost
+    immediately within the <guimenu>Dashboard</guimenu>.
+   </para>
+  </sect3>
+ <sect3 xml:id="sec.ha.geo.manage.hawk2.tickets">
+  <title>Managing Tickets with &hawk;2</title>
+  <para>
+   <remark>taroth 2015-10-29: krig: is the following still true?</remark> Tickets are visible in
+   &hawk;2 if they have been granted or revoked at least once <!--check!--> or if they are
+   referenced in a ticket dependency. </para>
+  <para>&hawk;2 displays the following ticket states:</para>
```

Comment:
The "elsewhere" state is unfortunately missing right now, only Granted and Revoked are shown. This is a bug that will be fixed, hopefully in time for the release.